src/elfpy/pricing_models/hyperdrive.py

```diff
@@ -155,10 +161,10 @@ def calc_in_given_out(
             user_result=user_result,
             market_result=market_result,
             breakdown=TradeBreakdown(
-                without_fee_or_slippage=float(flat + Decimal(curve.breakdown.without_fee_or_slippage)),
-                without_fee=float(flat + Decimal(curve.breakdown.without_fee)),
-                fee=curve.breakdown.fee,
-                with_fee=float(flat + Decimal(curve.breakdown.with_fee)),
+                without_fee_or_slippage=float(flat_without_fee + Decimal(curve.breakdown.without_fee_or_slippage)),
+                without_fee=float(flat_with_fee + Decimal(curve.breakdown.without_fee)),
+                fee=curve.breakdown.fee + float(redemption_fee),
```

Comment:
This has the opposite pattern as the other parts. Everywhere else you say `float(flat + Decimal(curve))` but here you say `curve + float(flat)`. Is that a typo, if not why is it different here?

Reply:
Not sure I understand, both calc_in_given_out and calc_out_given_in follow the same pattern for returning the trade result

Reply:
Ah I see what you mean now from IRL convo, changing.